cmd/internal/env/command_test.go

```diff
@@ -30,3 +31,7 @@ func TestCommand(t *testing.T) {
 	assert.True(t, strings.Contains(buf.String(), "METRICS_BACKEND"))
 	assert.True(t, strings.Contains(buf.String(), "SPAN_STORAGE"))
 }
+
+func TestMain(m *testing.M) {
+	goleak.VerifyTestMain(m, goleak.IgnoreTopFunction("github.com/golang/glog.(*fileSink).flushDaemon"))
```

Comment:
> goleak.IgnoreTopFunction("github.com/golang/glog.(*fileSink).flushDaemon")

Let's move this to pkg/testutils (with comment explaining why it's needed) and reuse instead of repeating everywhere.

Reply:
yes, that would be better